File: Orion/Components/ImportExport/Interface/SiriusImport/RestContracts/BaseClaimProcessPerilReserveType.vb
Public Class BaseClaimProcessPerilReserveType
    Public Property IsPaidToDate As Boolean
    Public Property IsReserveToDate As Boolean
    Public Property TypeCode As String
    Public Property Amount As Decimal
    Public Property TaxGroupCode As String
    Public Property PaymentAmount As Decimal
    Public Property PaymentAmountSpecified As Boolean
    Public Property ReverseExcess As Boolean = False
    Public Property PaymentDetails As BaseClaimProcessPaymentDetailsType
End Class

File: SSP.PureInsuranceRestAPIHandler/SSP.PureInsuranceRestAPIHandler/BaseClasses/BaseClaimProcessPerilReserveType.cs
namespace SSP.PureInsuranceRestAPIHandler.BaseClasses
{
    public class BaseClaimProcessPerilReserveType
    {
        public bool IsPaidToDate { get; set; }
        public bool IsReserveToDate { get; set; }
        public string TypeCode { get; set; }
        public decimal Amount { get; set; }
        public string TaxGroupCode { get; set; }
        public decimal PaymentAmount { get; set; }

        public bool PaymentAmountSpecified { get; set; }
        public bool ReverseExcess { get; set; } = false;
        public BaseClaimProcessPaymentDetailsType PaymentDetails { get; set; }
    }
}
